content/millennium/r4/scheduling/appointment.md

```diff
@@ -81,10 +81,15 @@ Notes:
 - The `patient`, `practitioner`, and `location` parameters may be included only once and may not be used in combination.
   For example, `patient=4704007,1316024` is supported but `patient=4704007&patient=1316024` and `patient=4704007&location=633867` are not.
 
-- The `date` parameter may be provided:  
-  - once with a prefix and time component to indicate a specific date/time. (e.g. `&date=ge2019-12-07T22:22:16.270Z`, `&date=lt2019-12-14T22:22:16.270Z`)     
-  - twice with the prefixes `ge` and `lt` to indicate a specific range. The date and prefix pairs must define
-    an upper and lower bound. (e.g. `&date=ge2019-12-07T22:22:16.270Z&date=lt2019-12-14T22:22:16.270Z`)
+- The `date` parameter may be provided when using `patient`, `practitioner`, and `location`.  
+  - Can be provided twice with the prefixes `ge` and `lt` to indicate a specific range. The date and prefix pairs must define
+    an upper and lower bound. (e.g. `&date=ge2019-12-07T22:22:16.270Z`, `&date=lt2019-12-14T22:22:16.270Z`) 
+  - Can also be provided once with a prefix and time component.
```

Comment:
~~Can also be provided~~ once with a prefix and time component to imply a range according to the following logic:

Reply:
Fix: https://github.com/cerner/fhir.cerner.com/pull/298/commits/2bfdc3e21d5206c5a2345c4d490c685c3d072ab0#diff-1d7df8ceb9f0ae8471323516e037c662R87